### DISCUSSION TITLE
why is crablang/crabgo#1 merged but not crablang/crabgo#2 merged

### DISCUSSION THREAD

**Discussion by Cuteistfox** [crablang/crab#66] [Q&A] [answered]
crablang/crabgo#1 is for #20 .crab but crablang/crabgo#2 is for #53 (.🦀)

**Comment by trvswgnr** [comment, marked as answer]
would love to support both! going to prioritize .crab first though, because emojis break things.
https://github.com/crablang/crabgo/pull/2#issuecomment-1576147977

**Reply by Cuteistfox** [reply]
break what
